islandora_compound_object.module

```diff
@@ -613,41 +630,53 @@ function islandora_compound_object_retrieve_compound_info(AbstractObject $object
   $rels_predicate = variable_get('islandora_compound_object_relationship', 'isConstituentOf');
   $part_of = $object->relationships->get(FEDORA_RELS_EXT_URI, $rels_predicate);
   $info = array();
-  // This object is part of a compound object.
+
+  // This object is the child of a compound object.
   if (!empty($part_of)) {
     // Go through all parents.
+    // XXX: Handle the case where we are a member of multiple compounds?
     foreach ($part_of as $part) {
       $parent_pid = $part['object']['value'];
       $parent = islandora_object_load($parent_pid);
```

Comment:
Yes. I was just using the behaviour that we had before where we load each possible parent. In practice most times we will only have one parent. 

I submitted a new commit that just directly uses the last value in the array instead of bothering to iterate.